lib/core/BlogPostLayout.js

```diff
@@ -123,6 +123,11 @@ class BlogPostLayout extends React.Component {
               </a>
             </div>
           </Container>
+          {this.props.config.onPageNav == 'separate' && (
+            <nav className="onPageNav">
+              <OnPageNav rawContent={this.props.children} />
```

Comment:
@ericnakagawa I think OnPageNav is not imported ☺

`const OnPageNav = require('./nav/OnPageNav.js');`

Reply:
I believe @endiliey is correct. That's why the Circle test is failing. 

I am confused though how it would work for you without the `require` in your testing 😮

Reply:
Well that’s embarrassing!

@joel I modify my local npm-module usually. I’ll push a fix.